src/libs/E2E/apiMocks/openReport.js

```diff
@@ -6,91 +6,1969 @@ export default () => ({
             value: {
                 reportID: '98345625',
                 reportName: 'Chat Report',
+                type: 'chat',
                 chatType: '',
+                ownerEmail: '__fake__',
                 ownerAccountID: 0,
+                managerEmail: '__fake__',
```

Comment:
FYI we don't want to be storing `ownerEmail` or `managerEmail` anymore - I'm in the process of removing these from everywhere in the app & soon will stop sending them from the backend in onyx 👍